python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/resources.py

```diff
@@ -247,20 +254,32 @@ def load_specs(
             return [*asset_specs, *asset_check_specs]
 
     def load_asset_specs(
-        self, dagster_dbt_translator: Optional[DagsterDbtTranslator] = None
+        self,
+        dagster_dbt_translator: Optional[DagsterDbtTranslator] = None,
+        select: str = "fqn:*",
+        exclude: Optional[str] = None,
```

Comment:
yea really don't like how this is optional at every layer down the chain - feels error prone